mappings/net/minecraft/server/world/ServerWorld.mapping

```diff
@@ -41,7 +41,7 @@ CLASS uk net/minecraft/server/world/ServerWorld
 	METHOD m_ updateEntities ()V
 	METHOD p tickScheduledTicks ()V
 	METHOD r getForcedSpawnPoint ()Let;
-	METHOD s getChunkProvider ()Luj;
+	METHOD s getChunkManager ()Luj;
```

Comment:
Seems to be another duplicate here

Reply:
getChunkManager is part of the IWorld interface, and implemented in both World and ServerWorld. In the interface and in World, it returns a ChunkManager. In ServerWorld it returns a ServerChunkManager. I'm pretty sure one of the above is a synthetic bridge method returning a ChunkManager